### PR TITLE
Not fully generated structure (New Year Tile) :D

### PR DESCRIPTION
**Pull request by Beta-King**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  **Here you can add your preview link--**
- [Preview Link] http://rawgit.com/Beta-King/labyrinth/master/index.html
- Fixes`#<Issue no.>`
, 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]

### My steps to solve it,
[Mention the files changed. Add what changes you have done.] 

### Screenshots,
[If any]

### CODE REVIEW

**Comment by Beta-King** [comment]
please review and merge :D

File: js/tiles.js [hunk 1]
```diff
@@ -781,12 +781,14 @@ const door = {
...
             this.wallRight = this.createImage("tiles/rooms/wall/rightNewYear.svg");
             this.ground = this.createImage("tiles/rooms/floor/floorNewYear.svg");
+            this.wallTop = this.createImage("tiles/rooms/wall/top.svg");
         },
         visit: function(player) {
         	playAudio("celebrate.mp3");
             this.wallTop.show();
             this.wallRight.show();
             this.ground.show();
+            this.wallTop.show();
```
Comment (duttaditya18) [inline js/tiles.js]: This was already there. This is not required to add it again.

File: js/tiles.js [hunk 2]
```diff
@@ -781,12 +781,14 @@ const door = {
             this.wallTop = this.createImage("tiles/rooms/door/topNewYear.svg");
```
Comment (duttaditya18) [inline js/tiles.js]: Remove this.

**Comment by Beta-King** [comment]
no i was trying to put the wall in the bottom @duttaditya18 as it is missing right ??

**Comment by Beta-King** [comment]
Ohk nevermind, i saw my mistake...

**Pull request closed without merging**